bin/build

```diff
@@ -28,10 +25,11 @@ if [ -n "$LOCAL_RPM" ] && [ -n "$REBUILD_RPM" ]; then
   exit 1
 fi
 
-REPO=${REPO:-docker.io/manageiq}
 BASE_DIR=${BASE_DIR:-$PWD}
+CONTAINER_PREFIX=${CONTAINER_PREFIX:-manageiq}
```

Comment:
I assume we'll start using this in the future?

Reply:
Yes, but for now we can at least have the same options.